IOPool/Input/src/RootFile.cc

```diff
@@ -638,7 +642,7 @@ namespace edm {
   }
 
   std::unique_ptr<FileBlock>
-  RootFile::createFileBlock() const {
+  RootFile::createFileBlock() {
```

Comment:
@Dr15Jones Good catch! The FileBlock contains a shared_ptr&lt;BranchChildren&gt;. By changing it to shared_ptr&lt;BranchChildren const&gt;, I was able to make createFileBlock() const again. This will be in the next commit.